packages/wallets/src/components/WalletTourGuide/WalletTourGuide.scss

```diff
@@ -0,0 +1,66 @@
+.wallets-tour-guide {
+    &__container {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        align-self: stretch;
+        border-radius: 0.8rem;
+        background: var(--system-light-8-primary-background, #fff);
+        width: 28rem;
+
+        /* Shadows/xxxl */
+        box-shadow: 0px 32px 64px 0px rgba(14, 14, 14, 0.14);
+    }
+
+    &__header {
+        display: flex;
+        height: 4.8rem;
+        padding: 0.8rem 0.8rem 0.8rem 1.6rem;
+        align-items: center;
+        gap: 1rem;
+        align-self: stretch;
+
+        border-radius: 0.8rem 0.8rem 0rem 0rem;
+        background: var(--system-light-8-primary-background, #fff);
+
+        /* divider/light/bottom/1px */
+        box-shadow: 0px -1px 0px 0px #f2f3f4 inset;
+    }
+
+    &__title {
+        flex: 1 0 0;
+
+        color: var(--brand-coral, #ff444f);
+        /* desktop/paragraph/P2 - bold */
+        font-size: 1.4rem;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 2rem; /* 142.857% */
+    }
```

Comment:
Dont think you'll be needing this

Reply:
@thisyahlen-deriv what exactly do you mean? All properties for title?

Reply:
Can use WalletText for this since there are styles for text here

Reply:
@thisyahlen-deriv there are different red colors, `WalletText` uses `#ec3f3f`, but we need `#ff444f` for title

Reply:
We can use the existing red in wallettext for now. Will add in the variations soon 😸

Reply:
@thisyahlen-deriv I updated the PR, check the last commit please 🙏